src/app/containers/CpsMetadata/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useContext } from 'react';
+import { string } from 'prop-types';
+import { ServiceContext } from '#contexts/ServiceContext';
+import Metadata from '../Metadata';
+import { getBrandedImage } from './utils';
+
+const CpsMetadata = ({
+  title,
+  language,
+  description,
+  firstPublished,
+  lastPublished,
+  indexImage,
+  indexImageAltText,
+}) => {
+  const { service } = useContext(ServiceContext);
+  const brandedImage = indexImage ? getBrandedImage(indexImage, service) : null;
+  return (
+    <Metadata
+      title={title}
+      lang={language}
+      description={description}
+      openGraphType="article"
+      openGraphImage={brandedImage}
+      openGraphImageAltText={indexImageAltText}
+    >
+      <meta name="article:author" content="https://www.facebook.com/bbcnews" />
```

Comment:
This should not be hardcoded, as it can differ from service to service - it should be passed in as articleAuthor (from ServiceContext).  E.g. https://github.com/bbc/simorgh/blob/cf7916fc34f067dd4289eb41ca994f86138d3041/src/app/lib/config/services/arabic.js#L15

Reply:
`<meta name="article:author" content={articleAuthor}/> `